### ISSUE TITLE
Digitalstrom: get zone group ID data type error

### ISSUE THREAD

**Issue by maggu2810** [eclipse/smarthome#4081]
The function in `org.eclipse.smarthome.binding.digitalstrom.internal.lib.manager.impl.StructureManagerImpl` currently looks like:

```Java
    @Override
    @SuppressWarnings("unchecked")
    public short getZoneGroupId(String zoneName, String groupName) {
        if (this.zoneGroupNameIdMap == null) {
            return -1;
        }
        if (this.zoneGroupNameIdMap.get(zoneName) != null) {
            if (this.zoneGroupNameIdMap.get(zoneName)[1] != null) {
                if (((HashMap<Short, String>) this.zoneGroupNameIdMap.get(zoneName)[1]).get(groupName) != null) {
                    return ((HashMap<String, Short>) this.zoneGroupNameIdMap.get(zoneName)[1]).get(groupName);
                }
            }
        }
        return -1;
    }
```

Eclipse IDE reports:

```text
Unlikely argument type String for get(Object) on a Map<Short,String>
```

So, it is clear that `HashMap<Short, String>` needs to be changed to `HashMap<String, Short>`.

But I don't think the `Object[]` etc. should be used at all.
I would prefer fixed types.

**Comment by triller-telekom** [comment]
For reference this fix is included in #3182 and thus this issue can be closed once PR #3182 has been merged.

**Comment by triller-telekom** [comment]
Since #3182 has been merged and 

https://github.com/eclipse/smarthome/blob/dcef6131b70c231a604b4beec5dc9e1769fbb88f/extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/manager/impl/StructureManagerImpl.java#L179-L186

does not contain this cast anymore I guess this issue can be fixed? @maggu2810 @MichaelOchel
